samples/sampler/stories/default/Dropdown.js

```diff
@@ -33,3 +32,56 @@ storiesOf('Agate', module)
 			text: 'The basic Dropdown'
 		}
 	);
+
+storiesOf('Agate QA.Dropdown', module)
+	.add(
+		'Dropdown directions',
+		() => {
+			const itemCount = number('items', Config, {range: true, min: 0, max: 50}, 5);
+			const items = (new Array(itemCount)).fill().map((i, index) => `Option ${index + 1}`);
+
+			return (
+				<div>
+					<div style={{height: '350px'}}>
```

Comment:
Please do not use pixel unit in samplers. I recommend you to use ri.scaleToRem() instead of `px`.

Reply:
done